packages/wdio-cli/src/interface.ts

```diff
@@ -356,6 +357,7 @@ export default class WDIOCLInterface extends EventEmitter {
             snapshotNotes.forEach((note) => this.log(note))
         }
 
+        log.info('Duration:\t', chalk.green(duration.getSummary()))
```

Comment:
Can you provide an example output of the `wdio.log` file?

Reply:
```
2025-08-16T12:02:24.021Z INFO @wdio/cli: Duration:	 12.6s (setup 5ms, prepare 392ms, execute 12.2s, complete 0ms)
```
It looks like this. It would be better to remove the `\t`. I'll remove it.

Reply:
```
2025-08-16T12:16:45.594Z INFO @wdio/cli: Duration: 12.2s (setup 4ms, prepare 243ms, execute 11.9s, complete 0ms)
```
How about this one